config/holes.toml

```diff
@@ -2536,14 +2538,18 @@ preamble = '''
 </div>
 
 <p>
-    Each input comprises of a (optional) prefix and one of the base or derived
-    units symbol, except that symbols for decimal multiples and sub-multiples
-    of "kg" are formed by attaching prefixes to the unit symbol "g". For
-    example, "10^−6 kg" is written as "mg", not as "µkg". Each such input
-    should be convert to product of a power of 10 and a product of base units,
-    as described in the tables above. If the unit is "1", only the power of
-    ten should be printed. Furthermore, "10^1" should be printed as "10" and
-    "10^0" as "1". For example, "MJ" should be converted to "10^6 kg m^2 s^-2".
+    Each argument generally contains a prefix and a unit, either base or derived,
+    with the exception of symbols representing decimal multiples and sub-multiples
+    of "kg" because those are formed by prepending the letter "g" with a prefix
+    symbol. For example: "10^−6 kg" results in printing "mg" instead of "µkg".
```

Comment:
Is it just me or does this make it sound a bit like "mg" is the output here when it actually is an input?